src/option_bytes.c

```diff
@@ -472,6 +472,44 @@ static int stlink_write_option_bytes_h7(stlink_t *sl, uint8_t *base,
   return 0;
 }
 
+/**
+ * Write option bytes
+ * @param sl
+ * @param addr of the memory mapped option bytes
+ * @param base option bytes to write
+ * @return 0 on success, -ve on failure.
+ */
+static int stlink_write_option_bytes_wb(stlink_t *sl, uint8_t *base,
+                                        stm32_addr_t addr, uint32_t len) {
+  /* Write options bytes */
+  uint32_t val;
+  int ret = 0;
+  (void)len;
+  uint32_t data;
+
+  clear_flash_error(sl);
+
+  write_uint32((unsigned char *)&data, *(uint32_t *)(base));
```

Comment:
WL5x chips have size of option bytes area of a few words. I think it is better to add a loop write by word similar to H7 chips: https://github.com/stlink-org/stlink/blob/468b1d2daa853b975c33ab69876c486734f2c6a7/src/option_bytes.c#L424

Reply:
Added the while loop